src/containers/CoursesPanel/CourseList/index.test.jsx

```diff
@@ -7,11 +7,15 @@ jest.mock('./hooks', () => ({
   useIsCollapsed: jest.fn(),
 }));
 
-jest.mock('containers/CourseCard', () => 'CourseCard');
+jest.mock('containers/CourseCard', () => jest.fn(() => <div>CourseCard</div>));
 jest.mock('containers/CourseFilterControls', () => ({
-  ActiveCourseFilters: 'ActiveCourseFilters',
+  ActiveCourseFilters: jest.fn(() => <div>CourseFilterControls</div>),
```

Comment:
Should this instead be:
```suggestion
  ActiveCourseFilters: jest.fn(() => <div>ActiveCourseFilters</div>),
```

since `CourseFilterControls` is the name of another component?

Reply:
done!